applications/CompressiblePotentialFlowApplication/custom_processes/define_embedded_wake_process.cpp

```diff
@@ -0,0 +1,143 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:        BSD License
+//                  Kratos default license: kratos/license.txt
+//
+//  Main authors:   Marc Nunez
+//
+
+
+#include "define_embedded_wake_process.h"
+#include "processes/calculate_discontinuous_distance_to_skin_process.h"
+#include "compressible_potential_flow_application_variables.h"
+#include "custom_utilities/potential_flow_utilities.h"
+
+
+namespace Kratos
+{
+// Constructor for DefineEmbeddedWakeProcess Process
+DefineEmbeddedWakeProcess::DefineEmbeddedWakeProcess(ModelPart& rModelPart,
```

Comment:
FYI, regarding this file, I considered inheriting from the body_fitted at first, but in the end both the construction and the actual process are fairly different.